index.html

```diff
@@ -1363,20 +1363,18 @@
 the core concepts of that domain combine into a [=user=]-comprehensible [=party=], and those refined
 definitions are likely to differ between domains.
 
-The <dfn data-lt="first parties">first party</dfn> is a [=party=] with which a [=person=] intends to
-interact. Merely hovering over, muting, pausing, or closing a given piece of content does
-not mean a [=person=] intends to interact with another party, nor does the simple
-fact of loading a [=party=] embedded in the one with which the person intends to
-interact. In cases of clear and conspicuous joint branding, there can be multiple [=first
-parties=]. The [=first party=] is necessarily a [=data controller=] of the data processing
-that takes places as a consequence of a [=person=] interacting with it.
-
-<aside class="issue">Refine the description of third parties so it matches
-intuition better.</aside>
-
-A <dfn data-lt="third parties">third party</dfn> is any [=party=] other than the [=person=] visiting the website,
-the [=first party=], or a [=service provider=] acting on behalf of either the [=person=] or
-the [=first party=].
+The [=parties=] that a reasonable person would think they are interacting with are known
+as the <dfn id="dfn-first-party" data-lt="first party">first parties</dfn>. Usually, there
+is just one first party, but clear-enough joint branding can make people aware that
+they're interacting with multiple other parties. When a [=person=] hovers over, mutes,
```

Comment:
Can we just skip mentioning the rare case of an interaction with multiple first parties?